### PR TITLE
feat(dapp-console-api): add api job queue and worker

### PR DESCRIPTION
**Pull request by jakim929**
- adds a new Processor service that initiates the worker, with a different entrypoint `runProcessor`
- sets up Dockerfile build for the Processor service + circleCI build
- sets up local docker compose
- adds an example job `sayHello` that is called once by the api server

### CODE REVIEW

**Comment by jakim929** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ecosystem/382?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#382** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ecosystem/382?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#364** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ecosystem/364?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jakim929 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dapp-console-api/src/constants/envVars.ts [hunk 1]
```diff
@@ -88,6 +88,7 @@ const envVarSchema = z.object({
   FAUCET_ACCESS_ATTESTERS: z.string().array(),
   WORLDID_APP_ID: z.string(),
   WORLDID_APP_ACTION_NAME: z.string(),
+  API_QUEUE_REDIS_URL: z.string(),
```
Comment (tremarkley) [inline apps/dapp-console-api/src/constants/envVars.ts:91]: nit: can you add this env var to `example.env`?

**Review by tremarkley — APPROVED** [review]
nicely done!

File: apps/dapp-console-api/src/constants/envVars.ts (hunk 1 above)
Comment (jakim929) [reply apps/dapp-console-api/src/constants/envVars.ts:91]: will fix!

**Pull request merged**